changelogs/unreleased/no-gh-add-tt-dependency.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
+
+* Added tt weak dependency
```

Comment:
I would like to see an issue with the motivation behind the change.

Reply:
Issue reference is added.